apps/sap-commerce-cloud/frontend/src/components/Dialog/Dialog.tsx

```diff
@@ -39,19 +39,16 @@ const Dialog: React.FC<DialogProps> = ({ sdk }) => {
   const [selectedProducts, setSelectedProducts] = useState<string[]>([]);
   const [errors, setErrors] = useState<ErrorType[]>([]);
 
-  const load = useCallback(() => {
-    const load = async () => {
-      const { products, errors } = await fetchProductList(
-        baseSite,
-        query,
-        page,
-        sdk.parameters as SAPParameters,
-        setTotalPages
-      );
-      setProducts(products);
-      setErrors(errors);
-    };
-    return load;
+  const load = useCallback(async () => {
```

Comment:
The Dialog tests were failing since this component wasn't actually loading since we weren't calling the load function